CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.250 Jan 12 2023
+- Add `NodePoolStatus` type to the `NodePool` resource model.
```

Comment:
We usually add these during the release tagging, after the actual changes have been merged, so you should probably remove this change.